src/OrbitClient.php

```diff
@@ -201,6 +201,7 @@ private function getRequestHeaders(array $options)
         }
         if ($this->options['env'] === 'stage') {
             $headers['x-forge-environment'] = 'stage';
+            $headers['x-feature'] = 'forge-free';
```

Comment:
In line 200 the x-feature header could be set as well, this mean that line 204 could override the value, you probably want to check first if is empty and if not you can appending to the already exising string, example:
```
if (empty($headers['X-Feature'])) {
    $headers['x-feature'] = 'forge-free';
} else {
   $headers['x-feature'] .= ', forge-free'
}
```

This code is just an example, it could be wrong, I don't remember if arrays key are case sensitive and I also don't remember if comma sepparated values is good enough for adding multiple values to a header.

Reply:
Ahhh great point, thanks Tasio :) - shall do some investigating 🕵️